modules/administration/pages/security-hardening.adoc

```diff
@@ -0,0 +1,54 @@
+[[security-hardening]]
+= Security Hardening
+
+While no implementation is ever completely secure from bad actors, there are some things you can do within {productname} to improve your security.
+
+This section contains a number of suggestions for hardening your security posture.
+The primary focus is to reduce the likelihood that a bad actor could access Salt clients in the case of a breach of the {productname} Server.
+In this case, we assume that the Salt master is running on the {productname} Server.
+
+To begin, there are some built-in mechanisms that make the {productname} Server less vulnerable to bad actors.
+The Salt master runs as a non-privileged user, and the Salt API only listens on localhost, not an external address.
+
+Additionally, Salt clients cannot send jobs to the Salt master in the default configuration.
+This can be manually adjusted, but it is not recommended.
+
+There are a limited number of ports required for clients to communicate with the server, which simplifies network management.
+The only required ports are 4505 and 4506.
+Ports 80 and 443 are only used for package repositories, and they secured with SSL trusted certificates and GPG keys.
+
+The server never initiates a TCP handshake with a client.
+Clients initiate every connection, and the communication is bi-drectional after initiation.
+
+Each client creates a set of RSA keys.
+The public key is accepted by the server when a new system is added.
+If the keys are secure on the server, only clients with the correct private keys can be authenticated.
+The keys should be secured by specific users.
+
+For extra key security, you can implement a key policy.
+A key policy that allows only pre-seeded keys on the master is the most secure.
+Additionally, you can implement a key policy that allows clients to communicate only with a master that has a specific digital fingerprint.
```

Comment:
I have no idea what this mean and I think customers will have the same question.

How to implement "a key policy"?
How to set it up/configure it?

If this is a guide which should explain how to make a server more secure, we need examples for everything!

As a sales guy say: You need to provide examples for everything. If you don't, customers become creative.